definitions/tools/check_strand.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "how_are_we_stranded_here"
+baseCommand: ['check_strandedness']
+arguments: [
+    "--print_commands"
+]
+
+stdout: "$(inputs.reads1.nameroot).strandness_check.txt"
```

Comment:
I might call this file `strandedness_check` to better match the name of the command.